zipline/_protocol.pyx

```diff
@@ -747,6 +747,12 @@ cdef class BarData:
         def __set__(self, val):
             self._adjust_minutes = val
 
+    property current_session:
+        def __get__(self):
+            return self._trading_calendar.minute_to_session_label(
```

Comment:
Can we explicitly pass `direction='next'` here to clarify the behavior in `before_trading_start`?